splash/tests/test_execute_emulation.py

```diff
@@ -60,21 +60,15 @@ class EmulatedMetaRedirectTest(Base.EmulationMixin, test_redirects.MetaRedirectT
 class EmulatedRenderPngTest(Base.EmulationMixin, test_render.RenderPngTest):
     script = 'main = require("emulation").render_png'
 
-    # TODO: default width and height are not applied
-    @pytest.mark.xfail
     def test_ok(self):
```

Comment:
These tests just call super() methods; they were introduced in order to have something to mark with xfail. I think it is safe to delete these methods - parent's methods will be discovered and executed directly.